docs/sources/flow/reference/components/pyroscope.scrape.md

```diff
@@ -387,7 +388,8 @@ Name | Type | Description | Default | Required
 `delta` | `boolean` | Whether to scrape the profile as a delta. | `false` | no
 
 When the `delta` argument is `true`, a `seconds` query parameter is
-automatically added to requests. The `seconds` used will be equal to `scrape_interval - 1`.
+automatically added to requests. The `seconds` used will be, by default to `scrape_interval - 1`
+, or `profiling_duration` if specified. 
```

Comment:
```suggestion
automatically added to requests. The default value for the `seconds` query parameter is `scrape_interval - 1`. If you set `profiling_duration`, then `seconds` has the same value as `profiling_duration.  For example, if you set `scrape_interval` to `"15s"`, `seconds` defaults to `14s`. If you set `profiling_duration` to `16s`, then `seconds is set to `16s` regardless of the `scrape_interval` value.
```

This probably isn't the final wording. I'm trying to understand the logic first... is this close?

Reply:
Yes, that's the correct logic. I think your suggestion is sufficient but alas, there's a missing backtick here :D

<img width="363" alt="image" src="https://github.com/grafana/agent/assets/41283691/904ac18c-df63-49ee-a988-47c29b7f1ed2">

Reply:
I've updated the suggestion a little. I fixed the missing backtick, and adjusted the wording slightly.. and included semantic linebreaks.